docker/Dockerfile

```diff
@@ -32,12 +33,21 @@ RUN apk update && \
       yaml-dev \
       zlib-dev \
       ncurses-dev \
-      bison \
-      autoconf \
-    && echo "gem: --no-ri --no-rdoc" > /etc/gemrc \
-    && bundle install --system $BUNDLER_ARGS \
-    && apk del .ruby-builddeps \
-    && rm -rf /var/cache/apk/*
+    && echo "gem: --no-ri --no-rdoc" > /etc/gemrc
```

Comment:
We should leave this all in one command to use docker build caching. If you split this up to multiple commands this can cause some issues in the final image otherwise you always rebuild them completely